src/inspector/models/score/scoreaccessibilitysettingsmodel.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_INSPECTOR_SCOREACCESSIBILITYSETTINGSMODEL_H
+#define MU_INSPECTOR_SCOREACCESSIBILITYSETTINGSMODEL_H
+
+#include "models/abstractinspectormodel.h"
+#include "models/propertyitem.h"
+#include "async/asyncable.h"
+#include "context/iglobalcontext.h"
+#include "engraving/style/scorestylepreset.h"
+
+namespace mu::inspector {
+class ScoreAccessibilitySettingsModel : public AbstractInspectorModel
+{
+    Q_OBJECT
+    INJECT(mu::context::IGlobalContext, globalContext)
+    Q_PROPERTY(PropertyItem * scoreStylePreset READ scoreStylePreset WRITE setScoreStylePreset NOTIFY scoreStylePresetChanged)
```

Comment:
`PropertyItem*` properties are supposed to be constant:
```suggestion
    Q_PROPERTY(PropertyItem * scoreStylePreset READ scoreStylePreset CONSTANT)
```
Why? The `PropertyItem*` is not the _value_ of the property, but an _object_ that has own properties for the value, default value, etc. 
So, the properties of that object can change and can be modified from QML, so _those_ have NOTIFY and WRITE functions. But that object itself stays constant.